src/Messaging/src/RabbitMQ/Support/ShutdownSignalException.cs

```diff
@@ -8,18 +8,6 @@ namespace Steeltoe.Messaging.RabbitMQ.Support;
 
 public class ShutdownSignalException : Exception
 {
-    public ushort ClassId => Args.ClassId;
```

Comment:
Were these not in use by anything?

Reply:
There weren't used. Their values were already accessible by the `Args` argument directly (which isn't being checked for `null`, so the properties may throw), so I thought I'd remove these. This releaves us from having to remember to update these properties in case the structure of `RC.ShutdownEventArgs` changes, which is in an external library.